source/extensions/filters/network/connection_limit/BUILD

```diff
@@ -0,0 +1,39 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+envoy_cc_library(
+    name = "connection_limit_lib",
+    srcs = ["connection_limit.cc"],
+    hdrs = ["connection_limit.h"],
+    deps = [
+        "//include/envoy/event:dispatcher_interface",
+        "//include/envoy/network:filter_interface",
+        "//include/envoy/runtime:runtime_interface",
+        "//include/envoy/stats:stats_macros",
+        "//source/common/protobuf:utility_lib",
+        "//source/common/runtime:runtime_lib",
+        "@envoy_api//envoy/extensions/filters/network/connection_limit/v3:pkg_cc_proto",
+    ],
+)
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    category = "envoy.filters.network",
+    security_posture = "robust_to_untrusted_downstream",
```

Comment:
The CI error suggests this should be `robust_to_untrusted_downstream_and_upstream`.

Reply:
Thanks @rojkov, changed it to `robust_to_untrusted_downstream_and_upstream`.